evap/evaluation/tests/test_tools.py

```diff
@@ -62,20 +62,26 @@ def test_log_exceptions_decorator(self, mock_logger, __):
 
 
 class TestHelperMethods(WebTest):
-    def test_is_many_prefetched(self):
-        evaluation = baker.make(Evaluation)
+    def test_is_prefetched(self):
```

Comment:
Funnily, `test_logic` is typechecked within, but the calls to it are not, as this does not have `-> None`; doesn't matter to me, but if you want to go all the way, this should probably be added